tests/integration/reporter.spec.js

```diff
@@ -16,8 +16,8 @@ describe('runs tests on cloud', function () {
     expect(process.env.SAUCE_ACCESS_KEY).toBeDefined();
 
     const cucumberRunCommand = 'npx cucumber-js'
-    const format = path.join(process.cwd(), 'src/reporter.ts')
-    const args = `--config tests/integration/cucumber.json --format ${format} --publish-quiet`
+    const format = path.join(process.cwd(), 'src/reporter.js')
+    const args = `--config tests/integration/cucumber.json --format ${format}`
```

Comment:
`--publish-quiet` has been deprecated in 9.4.0 and will be removed in 11.0.0 or later. 

https://github.com/cucumber/cucumber-js/blob/842099b87b6d689c2a45f13c820fd382108346a8/docs/deprecations.md#publishquiet